src/main/java/com/minecolonies/colony/buildings/BuildingTownHall.java

```diff
@@ -26,7 +26,7 @@ public String getSchematicName()
     @Override
     public int getMaxBuildingLevel()
     {
-        return 2;
+        return 3;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Assign this magic number 3 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS109)